app/views/dashboards/_version.html.erb

```diff
@@ -4,19 +4,25 @@
   <td><%= version.user.email %></td>
   <td><%= version.created_at.to_formatted_s(:long_ordinal) %></td>
   <td>
-    <%= link_to 'Show GIBCT', version.gibct_link, target: "_blank",
-        class: "btn btn-success btn-xs", role: "button" %>
-    <% unless version.production? %>
-      <%= link_to 'Download Export CSV', dashboard_export_path('Institution'),
-          class: "btn btn-warning btn-xs", role: "button" %>
-      <% if version.publishable? %>
-        <%= link_to 'Publish to Production', dashboard_push_path, method: :post,
-            data: { confirm: "Are you sure you wish to publish preview version #{version.number} to production?" },
-            class: "btn btn-danger btn-xs",
-            role: "button" %>
-        <% else %>
-          <a class="btn btn-danger btn-xs" disabled role="button">Publish to Production</a>
-        <% end %>
+    <% if version.generating? %>
+        <i class="fa fa-gear fa-spin" style="font-size:24px"></i>
+        Reload page to refresh preview version status
+    <% else %>
+      <%= link_to 'Show GIBCT', version.gibct_link, target: "_blank",
+          class: "btn btn-success btn-xs", role: "button" %>
+        <%= link_to 'Download Export CSV', dashboard_export_version_path(version.number),
+            class: "btn btn-warning btn-xs", role: "button" %>
+      <% unless version.production? %>
```

Comment:
Suggestion to make these branching `if` statements easier to follow:
Remove this `unless` and it's corresponding `end`.  Then change the `else` below to `elsif version.preview?` (see comment below)

Reply:
Thanks!